### ISSUE TITLE
Week 8 ready for review

### ISSUE THREAD

**Issue by e-christenson** [e-christenson/homesteadProjectPlanner#5]
@pawaitemadisoncollege

**takeaways** ... I knew literally nothing about API's one week ago.  crash course to say the least.  It went well and I am happy to have one api in my project.   if its setup right....now that is another question.   

**challenges/problems**:  the API i chose takes a zip code and returns information that mapped to two objects.   so I had to use some odd looking calls like :

https://github.com/e-christenson/homesteadProjectPlanner/blob/f7bbc7255c2d7763a0453659d3cc9eaeac4c5ae6/src/main/java/hpp/project/planner/persistence/ZipApiDao.java#L34

seems bizarre ....but they work.  
until I figured out the above call I was not sure I was mapping both java classes objects with a single mapper call to one singel calss.  im my case I call mapper on ZipCode.java and it also maps PlacesItem.java.... the toString gave it away, it was calling data from the second class's ts so I knew the data was there.  

long story short, this api is getting me longitude and latitude, so I can use that in a free weather api.   I will be adding that in after checkpoint 3 

another week in the books :)

**Comment by pawaitemadisoncollege** [comment]
Great job implementing this zip code api into your project!  The need to merge the lat/long into a single value will probably depend on how the weather api takes in those values - as a single string or separate values. 

You might consider a method in the  ZipApiDao to return the full zip code entity, similar to how our GenericDAO returns a full entity object. You could let the caller get out of that object whatever values they need.
